### DISCUSSION TITLE
achievements

### DISCUSSION THREAD

**Discussion by Cardosofiles** [community/community#144128] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

em busca da conquista junto a comunidade dev

**Comment by Cardosofiles** [comment, marked as answer]
Buscando uma conquista

**Comment by Cardosofiles** [comment]
comment #01

**Comment by jesbalchiero** [comment]
Opa, para conseguir a conquista você deve conseguir realizar uma resposta em um tópico de discussão que seja classificado como resposta/solução para o problema apresentado por quem abriu um tópico de discussão. Espero ter esclarecido o processo para que você consiga ganhar a conquista desejada.
